packages/gql/src/balancer/pools.ts

```diff
@@ -84,7 +84,7 @@ export const poolSnapshotInRange = gql`
     poolSnapshots(
       where: { pool_in: [$poolId], timestamp_in: $timestamp }
       orderBy: timestamp
-      orderDirection: asc
```

Comment:
why change the direction here?

Reply:
Here: https://github.com/bleu-fi/balancer-tools/pull/430#discussion_r1364572679

when fetching `from` 17/10 `to` 18/10 it was returning day 17/10 first. Which resulted in catching 17/10 instead of 18/10 and using the dates wrongly by one day